packages/calcite-components/src/components/input-number/input-number.scss

```diff
@@ -223,6 +221,10 @@ input:focus {
   background-color: var(--calcite-input-actions-background-color, var(--calcite-color-foreground-1));
   border-inline-start-width: theme("borderWidth.0");
 
+  calcite-icon {
+    color: var(--calcite-input-actions-icon-color, var(--calcite-color-text-3));
```

Comment:
Instead of setting the color should it set `--calcite-icon-color`? I'm not sure it matters much as they both do the same but we do have a var for it. Applies to other places where we are setting the `color` instead of `--calcite-icon-color`

Reply:
Good Point. Though it is same in terms of implementation, I think we can choose one as convention and be consistent. 

- Setting token would make it independent of implementation inside icon component but susceptible to any naming changes.
- Using color property can cause regression if icon changes its implementation by removing the color property on the :host _(which is highly unlikely)_ &  changing the default value.

I am also leaning towards using tokens here. Should we add this convention in tokens wiki?

Reply:
> I am also leaning towards using tokens here. Should we add this convention in tokens wiki?

I would think so. It allows for another level of abstraction. @jcfranco thoughts?

Reply:
Agreed on overriding the subcomponent's token. This pattern is covered under the component tokens wiki: https://github.com/Esri/calcite-design-system/wiki/tokens-component#expanded-example-1.